acl_loader/main.py

```diff
@@ -765,6 +822,33 @@ def incremental_update(self):
                 for namespace_configdb in self.per_npu_configdb.values():
                     namespace_configdb.set_entry(self.ACL_RULE, key, self.rules_info[key])
 
+    def update_time_based_acl_rules(self):
+        """
+        Perform update of dynamic ACL rule.
+        Renew the TTL of ACL rule if existing, or insert new dynamic ACL rule
```

Comment:
Update the `start_time` and `end_time` if existing, or insert new time-based ACL rule

Reply:
Accept and thanks!